save-common/src/commonMain/kotlin/org/cqfn/save/core/plugin/PluginConfig.kt

```diff
@@ -62,13 +69,40 @@ data class GeneralConfig(
         } ?: this.tags
 
         return GeneralConfig(
-            this.execCmd,
+            this.execCmd ?: other.execCmd,
             mergedTag,
-            this.description,
-            this.suiteName,
+            this.description ?: other.description,
+            this.suiteName ?: other.suiteName,
             this.excludedTests ?: other.excludedTests,
             this.includedTests ?: other.includedTests,
             this.ignoreSaveComments ?: other.ignoreSaveComments
         )
     }
+
+    override fun validateAndSetDefaults(): GeneralConfig {
+        requireNotNull(execCmd) {
+            errorMsgForRequireCheck("execCmd")
+        }
+        requireNotNull(tags) {
+            errorMsgForRequireCheck("tags")
+        }
+        requireNotNull(description) {
+            errorMsgForRequireCheck("description")
+        }
+        requireNotNull(suiteName) {
+            errorMsgForRequireCheck("suiteName")
+        }
+        return GeneralConfig(
+            execCmd,
+            tags,
+            description,
+            suiteName,
+            excludedTests ?: "",
+            includedTests ?: "",
+            ignoreSaveComments ?: false
+        )
+    }
+
+    private fun errorMsgForRequireCheck(field: String) = "Error: Couldn't found `$field` in [general] section. Please provide it in this, " +
```

Comment:
```suggestion
    private fun errorMsgForRequireCheck(field: String) = "Error: Couldn't find `$field` in [general] section. Please provide it in this, " +
```
Also, can we specify, which config is `this` here?

Reply:
Fixed to `find`.
About specification - I was thinking about specifying the location, but in PluginConfig we have no such information, only in TestConfig.
I added at least printing of the fields of current config.

Reply:
please create an issue to add TestConfig field to the PluginConfig to support this kind of logging

Reply:
Created https://github.com/cqfn/save/issues/118